### PR TITLE
Parse github user mentions from strings

### PR DESCRIPTION
**Pull request by apelisse**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/contrib/1648)

### CODE REVIEW

File: mungegithub/mungers/mungerutil/util.go [hunk 1]
```diff
@@ -66,6 +76,20 @@ func (u UserSet) Mention() UserSet {
 	return UserSet(mentionedUsers)
 }
 
+func (u UserSet) UnMention() UserSet {
```
Comment (foxish) [inline mungegithub/mungers/mungerutil/util.go]: Where do we plan on using this?

Comment (apelisse) [inline mungegithub/mungers/mungerutil/util.go]: Depends how we implement the OWNERS file, let's not merge until we actually need it ...

**Comment by test-foxish** [comment]
recomputing cla status...

**Comment by grodrigues3** [comment]
[APPROVALNOTIFIER] The Following OWNERS Files Need Approval:

- **mungegithub**

We suggest the following people:
cc @lavalamp 
 You can indicate your approval by writing `/approve` in a comment
 You can cancel your approval by writing `/approve cancel`in a comment

**Comment by foxish** [comment]
Closing, since mungegithub doesn't live in contrib anymore anyway.

**Pull request closed without merging**